### ISSUE TITLE
postinstall error when installing using pnpm monorepo

### ISSUE THREAD

**Issue by btakita** [nolimits4web/swiper#4064]
When installing swiper using a pnpm monorepo, the following error occurs:

```
node_modules/.pnpm/swiper@6.4.4/node_modules/swiper: Running postinstall script, failed in 66ms
.../swiper@6.4.4/node_modules/swiper postinstall$ node postinstall.js
│ internal/modules/cjs/loader.js:883
│   throw err;
│   ^
│ Error: Cannot find module '/project/path/node_modules/.pnpm/swiper@6.4.4/node_modules/swiper/postinsta
│     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
│     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
│     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
│     at internal/main/run_main_module.js:17:47 {
│   code: 'MODULE_NOT_FOUND',
│   requireStack: []
│ }
└─ Failed in 66ms
 ERROR  Command failed with exit code 1.
```

**Comment by samizarraa** [comment]
Same error on update swiper
Error: Cannot find module 'path\node_modules\swiper\postinstall.js' 
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)        
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

**Comment by nolimits4web** [comment]
Fixed in 6.4.5
